dev-libs/nspr/patches/nspr-4.26.patchset

```diff
@@ -0,0 +1,324 @@
+From 392604e5ea00926197bf74e3ba484327264a131f Mon Sep 17 00:00:00 2001
+From: Ken Mays <kmays2000@gmail.com>
+Date: Sat, 27 Jun 2020 00:09:11 +0200
+Subject: Migrated NSPR 4.19 patchset to NSPR 4.26
+
+* NSPR 4.19 migration by Kacper Kasper
+* Old patchset wouldn't apply anymore.
+* Platform code by korli.
+* pr/include/obsolete/prototypes.h by pulkomandy.
+* Fix build of libpurple.
+* pr/include/prtypes.h by Kacper Kasper.
+  Fix build of NSS. NSS uses XP_BEOS and defs for BeOS fail on GCC7.
+
+diff --git a/configure.in b/configure.in
+index b62f1cd..0687af4 100644
+--- a/configure.in
++++ b/configure.in
+@@ -1483,6 +1483,30 @@ tools are selected during the Xcode/Developer Tools installation.])
+     PR_MD_CSRCS=freebsd.c
+     ;;
+ 
++*-haiku*)
++    AC_DEFINE(XP_UNIX)
++    AC_DEFINE(XP_HAIKU)
```

Comment:
I think we need to remove this line (and adjust references to it), this was only added after nspr version 4.19, prior to that it didn't exists, using !defined(__HAIKU__) for the PRUint(32) types fixed the build for xmlsec ...
```diff
- +    AC_DEFINE(XP_HAIKU)
```

Reply:
Done